command/agent/agent.go

```diff
@@ -1361,3 +1361,15 @@ func (a *Agent) DisableNodeMaintenance() {
 	a.RemoveCheck(nodeMaintCheckID, true)
 	a.logger.Printf("[INFO] agent: Node left maintenance mode")
 }
+
+// Query sends a Query on Serf, see Serf.Query.
```

Comment:
Query is such a generic term that it's easy to see some other part of the Consul layer cake using that term to mean something else entirely.  I think we should rename all uses `serfQuery` or `SerfQuery` as appropriate.